queries.sql

```diff
@@ -15,4 +15,48 @@ SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name != 'Gabumon';
 --Find all animals with a weight between 10.4kg and 17.3kg 
 --(including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+/*
+Inside a transaction update the animals table by setting the species column to unspecified.
+ Verify that change was made. Then roll back the change and verify that the species columns 
+ went back to the state before the transaction.
+*/
+--Start the Transactions
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'Digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species = 'Pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN TRANSACTION;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ Required ] I think it would be better if any time before doing a `Commit` or `Rollback` for a given `Transaction` it's good to always check if indeed the change was made. Please see the example below for more details on this and rectify.

## Example code
```SQL
BEGIN;

DELETE FROM animals; 
SELECT COUNT(*) FROM ANIMALS;  -- verify that change was made

ROLLBACK;
SELECT COUNT(*) FROM ANIMALS; -- verify that change persists after commit
```

## Project requirements
![Screenshot from 2023-08-24 02-32-53](https://github.com/Diegogagan2587/vet-clinic-database/assets/16392046/aee85f9d-4071-4dd4-a2b2-38351099ab5f)